pkg/ipamclient/errors.go

```diff
@@ -0,0 +1,19 @@
+package ipamclient
+
+import (
+	"errors"
+)
+
+var (
+	// ErrLBHTTPUnauthorized is returned when the request is not authorized
+	ErrLBHTTPUnauthorized = errors.New("ipam api received unauthorized request")
```

Comment:
should these be `ErrIPAMHTTP.....` (and below?)

Reply:
✅